integration-cli/docker_cli_run_unix_test.go

```diff
@@ -414,3 +414,19 @@ func (s *DockerSuite) TestTwoContainersInNetHost(c *check.C) {
 	dockerCmd(c, "stop", "first")
 	dockerCmd(c, "stop", "second")
 }
+
+func (s *DockerSuite) TestStopContainerSignal(c *check.C) {
```

Comment:
now that I think about, it doesn't make sense to have the platform check in the client because we must validate the signal in the daemon. Changing all this to move the checking to the server.